src/Spectre.libGaClassifier.Tests/Spectre.libGaClassifier.Tests.vcxproj

```diff
@@ -21,8 +21,8 @@
   </ItemGroup>
```

Comment:
Lots of changes in this file are unnecessary. Next time try to commit only necessary things, just to reduce the risk of conflicts during merge / rebase.

Reply:
ok, I will try